HerPublicWebsite.BusinessLogic/Models/LocalAuthorityData.cs

```diff
@@ -4,37 +4,37 @@ namespace HerPublicWebsite.BusinessLogic.Models;
 
 public class LocalAuthorityData
 {
-        public enum Hug2Status
-        {
-                NotTakingPart,
-                Pending,
-                Live
-        }
+    public enum Hug2Status
+    {
+            NotTakingPart,
+            Pending,
+            Live
+    }
```

Comment:
This is actually not quite the correct indentation yet, the individual values should be back another tab:

```
public class LocalAuthorityData
{
    public enum Hug2Status
    {
        NotTakingPart,
        Pending,
        Live
    }
```

but as per my overall comment, I think just leaving this alone for now is probably for the best. We can address it in your other reviews.